src/graph/topo.h

```diff
@@ -140,7 +145,6 @@ struct ncclTopoNode {
       int collSupport;
       int maxChannels;
       int64_t busId;
-      int dev;
```

Comment:
Since RCCL independently added the `dev` field, we need to verify that they are in fact "the same" field. Otherwise NCCL and RCCL changes will fight in this struct.